source/backend/routes/users.js

```diff
@@ -3,6 +3,8 @@ const usersModel = require('../database/models/usersModel');
 const completedRecipesModel = require('../database/models/completedRecipesModel');
 const savedRecipesModel = require('../database/models/savedRecipesModel');
 
+const bcrypt = require('bcrypt');
```

Comment:
use bcrypt-nodejs instead, bc the 2 libraries has different salt and hash, using different ones will be prone to error. especially when users trying to login.